### ISSUE TITLE
"openio container create" doesn't fail if the container exists

### ISSUE THREAD

**Issue by jfsmig** [open-io/oiopy#19]
Subsequent calls on container creation give no clue of something wrong. This is not a bug in itself, but it differ from previous commands.

    ### 0 /home/jfs
    jfs@localhost 52 # openio -v container create JFS ; echo $?
    command: container create -> oiopy.cli.storage.container.CreateContainer
    Using parameters {'proxyd_url': 'http://127.0.0.1:6002', 'namespace': 'NS', 'account_name': 'ACCT'}
    0
    ### 0 /home/jfs
    jfs@localhost 53 # openio -v container create JFS ; echo $?
    command: container create -> oiopy.cli.storage.container.CreateContainer
    Using parameters {'proxyd_url': 'http://127.0.0.1:6002', 'namespace': 'NS', 'account_name': 'ACCT'}
    0

At least, the "verbose" flag should display an informational message.

**Comment by jkasarherou** [comment]
What about something like that ?
```
oiopy - ❯ openio --oio-ns NS --oio-account myaccount container create 1 2 3 4
+-----------+-----------+---------+
| account   | container | created |
+-----------+-----------+---------+
| myaccount | 1         | False   |
| myaccount | 2         | True   |
| myaccount | 3         | False   |
| myaccount | 4         | True   |
+-----------+-----------+---------+
```

**Comment by jfsmig** [comment]
:+1:

**Comment by jkasarherou** [comment]
fixed in #24
